<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>交通路标识别系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;交通路标识别系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>191</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="button_my">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>个人信息</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="button_user">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="button_journal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>日志管理</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="button_system">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>系统管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_photo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
